views/index.tpl

```diff
@@ -131,6 +131,11 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
     </p>
   </div>
 
+  <div class="linkGen">
```

Comment:
What is class `linkGen` for? It seems no css style applied to this class, and javascript code using id `link` to set value of the text box.

If we didn't use this class, please remove it.